ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewManagersPropertyCache.java

```diff
@@ -184,11 +193,131 @@ public DoublePropSetter(ReactPropGroup prop, Method setter, int index, double de
     }
 
     @Override
-    protected Object getValueOrDefault(Object value) {
+    protected Object getValueOrDefault(Object value, Context context) {
       return value == null ? mDefaultValue : (Double) value;
     }
   }
 
+  private static class ColorPropSetter extends PropSetter {
```

Comment:
Do we have tests for other PropSetters on Android?

Reply:
I did find Java Unit Tests, but they don't build due to missing imports.   I tried to fix or comment out the missing imports and got them to build, but then only 12 of the 284 tests actually pass.   I think it would be trivial to add ColorPropSetter tests to `ReactPropAnnotationSetterTest` and perhaps `ReactPropForShadowNodeSetterTest` but getting the tests to work again are outside my abilities.  I'll see if @ArazAbishov can help with this.

Reply:
I made a commit to this PR that gets the Android tests to at least build again.  But getting them to pass again is a deep rabbit hole out of the scope of this change.   I made a separate branch that we're working on to get Android tests passing again but it should be an entirely separate Pull Request from this one.